manifests/profile/kubernetes/dns_server.pp

```diff
@@ -53,6 +54,12 @@
     ;
   }
 
+  $private_zones.each |Hash $zone| {
+    file { "/etc/dnsmasq.d/${$zone[name]}":
+      content => "server=/${$zone[domain]}/${$zone[resolver]}\n"
```

Comment:
Do you want this to notify the dnsmasq service? Or does it not need a restart when the configuration changes?

Reply:
Yes, it should notify the dnsmasq service -- good catch.